provisioning/service/src/Http/ContractApiHttp.cs

```diff
@@ -218,7 +218,7 @@ private static void ValidateHttpResponse(ContractApiResponse response)
                     if (response.StatusCode >= HttpStatusCode.Ambiguous)
                     {
                         throw new ProvisioningServiceException(
-                            $"{response.ErrorMessage}:{response.Body}",
+                            $"{response.ErrorMessage}:{responseBody.Message}",
```

Comment:
Using the message from the response for the exception message

Reply:
What's the difference here?

Reply:
```suggestion
                            $"{response.ErrorMessage}: {responseBody.Message}",
```

Reply:
`response.Body` is the complete unparsed response body, whereas `responseBody.Message` is the message body. 
Since we've parsed out the rest of the response into status, error code, tracking Id etc., it makes sense for the exception message to contain only the error message.